telemetry/definitions/commonDefinitions.json

```diff
@@ -12,9 +12,9 @@
         },
         {
             "name": "serviceType",
-            "allowedValues": ["ecs", "cloudformation", "lambda", "s3", "schemas"],
+            "allowedValues": ["ecs", "cloudformation", "lambda", "logs", "s3", "schemas"],
```

Comment:
"logs" is the name in the SDK?

Reply:
yep you can find these by doing a search for `SERVICE_NAME` in the Java SDK. Added that to the description as well